usaspending_api/awards/management/commands/helpers.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from usaspending_api.references.models import Location, RefCountryCode
+
+
+def up2colon(input_string):
+    'Takes the part of a string before `:`, if any.'
+
+    if input_string:
+        return input_string.split(':')[0].strip()
+    return ''
+
+
+def convert_date(date):
+    return datetime.strptime(date, '%m/%d/%Y').strftime('%Y-%m-%d')
+
+
+def fetch_country_code(vendor_country_code):
+    code_str = up2colon(vendor_country_code)
+
+    country_code = RefCountryCode.objects.filter(
+        Q(country_code=code_str) | Q(country_name__iexact=code_str)).first()
+    if not country_code:
+        # We don't have an exact match on the name or the code, so we need to
+        # chain filter on the name
+        query_set = RefCountryCode.objects
+        for word in code_str.split():
+            query_set = query_set.filter(country_name__icontains=word)
+        country_code = query_set.first()
+
+    return country_code
```

Comment:
Makes sense.  I'll just be aware that calling a string method on it might error, but I'm not doing any of that.  And our loaders are populating the database with a blank in those cases.  (I guess that relates a little to the comment above, because we could fill with NULLs instead.)